### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by yuletide**

### CODE REVIEW

File: accessibility/README.md [hunk 1]
```diff
@@ -1,16 +1,57 @@
...
 # Accessibility Tools
 
 These scripts are designed to be used to complete an isochrone based accessibility analysis for site selection, coverage, catchment area or microplanning analysis.
 
+Read more about why Mapbox built these tools and how to use them in this blog post
+
 ## Installation
 
-With [conda](https://docs.conda.io/en/latest/) installed, set up an environment with the following
+With [conda][1] installed, set up an environment with the following
```
Comment (mikelmaron) [inline accessibility/README.md:9]: why no link to conda @yuletide

File: accessibility/README.md [hunk 2]
```diff
@@ -1,16 +1,57 @@
...
-## Configuration
+## Usage
+
+To view detailed usage of each script run `python analyze.py --help` or `python isochrone.py --help`
+
+The first step is to run <code>[isochrones.py][2]</code>, a helper tool to generate one [isochrone][3] for each input point, and copy its attributes over to the output shape. Configure the script using the following parameters, which map to parameters [offered by the Isochrone API][4]:
+
+- Input: A geojson or csv files containing points to (*required*)
+- `token`: A Mapbox API token (*required*)
+- `output`: The filename to use for saving the output geometry. Default is inputfile_travelprofile_traveltime.json
```
Comment (mikelmaron) [inline accessibility/README.md]: is the output file pattern a template? it's not obvious from formatting here

File: accessibility/README.md [hunk 3]
```diff
@@ -1,16 +1,57 @@
...
+- `minutes`: The travel time in minutes to travel from the point. Default is 30m
+- `profile`: the mode of transport, driving, walking or cycling. Default is driving
+- `generalize`: a tolerance to use for simplifying the output geometries using the Douglas-Peucker algorithm, in meters. Use 0 (the default) to return the full isochrones, and to avoid occasional self-intersecting geometry errors
+- `limit`: If provided, the script will only read in the first *n* features of your input data. Useful for testing
+- `force`: Overwrite any existing output files with the same name. Otherwise, script will error if a file exists
+
+<code>python isochrones.py --help # for detailed usage info
```
Comment (mikelmaron) [inline accessibility/README.md]: duplicate of info in line 17

Comment (mikelmaron) [reply accessibility/README.md]: do we need to specify python3

File: accessibility/README.md [hunk 4]
```diff
@@ -1,16 +1,57 @@
...
+python isochrones.py --profile=driving --minutes=30 --generalize=0 --token=$MAPBOX_ACCESS_TOKEN sample_data/points.geojson</code>
+
+The output is a GeoJSON collection of all the isochrones for all points in the data sets. The next step is to run <code>[analyze.py][5]</code> which takes those isochrones and the original points, and calculates the total population within that area. Configure the script using the following parameters:
+
+- Input: A geojson file containing the isochrones generated in the previous step
+- `pop_tiff`: The path to a population raster file (*required*)
+- `buffer_distance`: The distance in meters used to buffer the input isochrones to reduce their geometric complexity and account for walking time to communities along the road. Default: 1
+- `limit`: If provided, only load a limited number of features, useful for testing small portions of a larger dataset
+- `output`: File name to save output geometry, specify a name with .gpkg to save as [Geopackage][6] (useful for visualizing in a GIS, or transferring to a database) or .geojson for Geojson (useful for tiling or visualizing with Mapbox GL JS)
```
Comment (mikelmaron) [inline accessibility/README.md]: GeoJSON

File: accessibility/README.md [hunk 5]
```diff
@@ -1,16 +1,57 @@
...
+- `points`: Input facility points, used for buffering to incorporate those living within direct walking distance, and missing roads
+- `points_buffer_distance`: The distance in meters to buffer the facility `points`. Default: 4000
+
+Next run <code>[analyze.py][5]</code>
+
+<code>python analyze.py --pop_tiff=sample_data/pop.tiff --points=sample_data/points.geojson --output=accessibility.geojson points_isochrone_driving_30.json</code>
+
+[1]: https://docs.conda.io/en/latest/
```
Comment (mikelmaron) [inline accessibility/README.md:47]: when not make all these links markdown links within the readme

**Review by mikelmaron — CHANGES_REQUESTED** [review]

File: accessibility/README.md (hunk 1 above)
Comment (yuletide) [reply accessibility/README.md:9]: Link is there -- this is husky enforcing the docs markdown styling rules

File: accessibility/README.md (hunk 3 above)
Comment (yuletide) [reply accessibility/README.md]: Depends on how your environment is setup -- but I think for most it defaults to python3 these days, which we specify in the requirements and env

File: accessibility/README.md (hunk 5 above)
Comment (yuletide) [reply accessibility/README.md:47]: These are docs markdown linting rules see https://github.com/mapbox/repo-roadie

**Pull request merged**